src/tezos_interop/fetch_storage.js

```diff
@@ -40,6 +41,14 @@ const error = (error) =>
   const { rpc_node, contract_address, confirmation } = input();
   const client = new RpcClient(rpc_node);
   const Tezos = new TezosToolkit(rpc_node);
+  Tezos.setProvider({
+    config: {
+      shouldObservableSubscriptionRetry: true,
```

Comment:
Since we use three times the same values, I would have extract them to commons const define somewhere relevant, in order to do something like
```javascript
# relevant place to define these const
const shouldObservableSubscriptionRetry = true;
const streamerPollingIntervalMilliseconds = 1000;

# Then within the other places:
  Tezos.setProvider({
    config: {
      shouldObservableSubscriptionRetry: shouldObservableSubscriptionRetry,
      streamerPollingIntervalMilliseconds: streamerPollingIntervalMilliseconds,
[...]
    },
  });
```
What do you think?

Reply:
We have quite a lot of shared code in those files, and they're being removed in the near future, I would like to avoid adding more infrastructure for them.

Reply:
Got it, maybe a simple comment like
> this is also defined in listen_transactions.js

to avoid any miss when we change/remove them?

Reply:
done boss